web/themes/custom/pineapple/sass/components/navigation/_breadcrumb.scss

```diff
@@ -0,0 +1,23 @@
+.block-system-breadcrumb-block {
+  .breadcrumb {
+    font-size: $small-font;
+    color: $dark-grey;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    padding-bottom: 0;
+
+    a {
+      color: $dark-grey;
+
+      &:hover,
+      &:focus {
+        color: $black;
+        text-decoration: underline;
+      }
+    }
+  }
+}
+
+.breadcrumb-divider {
+  border-bottom: 1px solid $light-grey;
+}
```

Comment:
Consider using a <hr> tag instead of creating a new class to style in this way

Reply:
done